docs/overview.md

```diff
@@ -1,27 +1,44 @@
 # Overview
 
-## What is jrnl?
+## What is `jrnl`?
 
-`jrnl` is a simple journal application for
-your command line. Journals are stored as human readable plain text
-files - you can put them into a Dropbox folder for instant syncing and
-you can be assured that your journal will still be readable in 2050,
-when all your fancy iPad journal applications will long be forgotten.
+`jrnl` is a simple but powerful plain text journal application for the command
+line.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+`jrnl` is not the only plain text journal application. It includes features that
+are common to many other applications:
+
+- Journal entries are created and displayed directly from the command line.
+- Journals are stored as human-readable, future-proof plain text files. Because
+  files are stored as plain text, you can rest assured that your journals will
+  still be readable in 2050 and beyond. Entries can be searched, filtered, and
+  edited rapidly and easily.
+- Journals can be stored anywhere, including in shared folders so that they can
+  be synchronized between devices.
+- Journal entries can be created and modified using an external editor of your
+  choice, using the command line as a starting point to filter specific entries.
+- BeJournals are stored in single files using date/time tags to identify
+  individual entries. Some systems endorse a file/folder methodology; `jrnl`
+  keeps 
+- Journals can be protected using [256-bit
```

Comment:
This whole bullet point sequence seems rather repetitive. Every line starts with journals. Maybe it's possible to rephrase some differently?